docs/examples/combobox/controlled.js

```diff
@@ -0,0 +1,137 @@
+// @flow strict
+import { useState, type Node } from 'react';
+import { Box, ComboBox, Flex, Text } from 'gestalt';
+
+export default function Example(): Node {
+  const US_STATES = [
```

Comment:
nit: these sort of things shouldn't be defined in components, as they'll be recreated on each render. not important here but a good thing to keep in mind

Reply:
DONE